launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -82,6 +87,22 @@ def __init__(
         self.__final_target_container_name = None  # type: Optional[Text]
         self.__logger = launch.logging.get_logger(__name__)
 
+    @classmethod
+    def parse(cls, entity: Entity, parser: Parser):
+        """Parse load_composable_node."""
+        _, kwargs = super().parse(entity, parser)
+
+        kwargs['target_container'] = entity.get_attr('target', data_type=str)
+
+        composable_nodes = entity.get_attr(
+            'composable_node', data_type=List[Entity], optional=True)
```

Comment:
this attribute shouldn't be `optional`

Reply:
So do you mean we don't need to support this use case(define empty container first, and load node later)?
I'm okay with that, but just a confirmation.

```xml
<launch>
  <node_container pkg="rclcpp_components" exec="component_container" name="my_container" namespace="" args="test_args">
  </node_container>

  <load_composable_node target="my_container">
    <composable_node pkg="composition" plugin="composition::Listener" name="listener" namespace="test_namespace">
    </composable_node>
  </load_composable_node>
</launch>
```

Since it's optional here https://github.com/ros2/launch_ros/blob/46bf3de8c995824784b57dd5e9b64e8872a40832/launch_ros/launch_ros/actions/composable_node_container.py#L37 , I thought it should be optional in frontend as well.

Reply:
@kenji-miyake it's optional for the `ComposableNodeContainer` action, as you say, but not for the `LoadComposableNode` action you're implementing a parser for right here:

https://github.com/ros2/launch_ros/blob/46bf3de8c995824784b57dd5e9b64e8872a40832/launch_ros/launch_ros/actions/load_composable_nodes.py#L55

Reply:
Oh, I understand! Thank you. :smile:

Reply:
Removed in 2248138.